apple/internal/linking_support.bzl

```diff
@@ -105,6 +110,7 @@ def _register_linking_action(ctx, extra_linkopts = []):
     return apple_common.link_multi_arch_binary(
         ctx = ctx,
         extra_linkopts = linkopts,
+        stamp = stamp,
```

Comment:
`link_multi_arch_binary() got unexpected keyword argument 'stamp'`

Reply:
Looks like this is a 3.8+ feature then